Editing Pod disruption budget

alaudazzi opened the pull request:
This PR suggests a few edits.

Relates to [#63068](https://github.com/elastic/cloud/issues/63068).

alaudazzi commented:
@barkbay I don't know why the file `/elasticsearch/init-containers-plugin-downloads.asciidoc` got automatically added to this PR. This file was already reviewed and merged in #3643.

charith-elastic commented on docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc:
```diff
@@ -27,6 +27,10 @@ spec:
 ----
 
-You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in the <<{p}-custom-images,custom images doc>>. The <<{p}-snapshots,snapshots>> document has more information on both these options. The Kubernetes document on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init containers] has more information on the ...
+You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in the <<{p}-custom-images,custom images doc>>. For more information on both these options, you can check the <<{p}-snapshots,Create automated snapshots>> section and the Kubernetes documentation on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init  ...
```
```suggestion
You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in <<{p}-custom-images>>. For more information on both these options, you can check <<{p}-snapshots>> section and the Kubernetes documentation on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init containers].
```

alaudazzi replied:
@charith-elastic the links formatting is weird in your suggestion. 
I propose to keep this version:

> You can also override the Elasticsearch container image to use your own image with the plugins already installed, as described in <<{p}-custom-images,custom images>>. For more information on both these options, you can check the <<{p}-snapshots,Create automated snapshots>> section and the Kubernetes documentation on https://kubernetes.io/docs/concepts/workloads/pods/init-containers/[init containers].

charith-elastic commented on docs/orchestrating-elastic-stack-applications/elasticsearch/init-containers-plugin-downloads.asciidoc:
```diff
@@ -27,6 +27,10 @@ spec:
+The init container inherits:
+
+* The image of the main container image, if one is not explicitly set.
+* The volume mounts, as long as the name and mount path do not conflict. 
```
```suggestion
* The volume mounts from the main container unless a volume mount with the same name and mount path is present in the init container definition.
```

charith-elastic commented on docs/orchestrating-elastic-stack-applications/elasticsearch/pod-disruption-budget.asciidoc:
```diff
@@ -8,12 +8,12 @@ endif::[]
 
-A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] allows limiting disruptions on an existing set of Pods while the Kubernetes cluster administrator manages Kubernetes nodes.
-Elasticsearch makes sure some indices don't become unavailable.
+A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] (PDB) allows you to limit disruptions on an existing set of Pods, while the Kubernetes cluster administrator manages Kubernetes nodes.
```
```suggestion
A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] (PDB) allows you to limit the disruption to your application when its pods need to be rescheduled for some reason such as upgrades or routine maintenance work on the Kubernetes nodes.
```

charith-elastic commented on docs/orchestrating-elastic-stack-applications/elasticsearch/pod-disruption-budget.asciidoc:
```diff
@@ -8,12 +8,12 @@ endif::[]
-A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] allows limiting disruptions on an existing set of Pods while the Kubernetes cluster administrator manages Kubernetes nodes.
-Elasticsearch makes sure some indices don't become unavailable.
+A link:https://kubernetes.io/docs/tasks/run-application/configure-pdb/[Pod Disruption Budget] (PDB) allows you to limit disruptions on an existing set of Pods, while the Kubernetes cluster administrator manages Kubernetes nodes.
+Elasticsearch makes sure that some indices don't become unavailable.
```
I don't think this line is necessary.

```suggestion

```

alaudazzi commented:
@charith-elastic I addressed the feedback you provided on `elasticsearch/init-containers-plugin-downloads.asciidoc` in a separate PR => #3647.  My apologies for the confusion.
